spec/light-client/supervisor/supervisor_001_draft.md

```diff
@@ -254,7 +254,7 @@ Replace_Secondary(addr Address, root-of-trust LightBlock)
     - maintain [LCD-INV-ROOT-AGREED.1], that is,
     ensure root-of-trust = FetchLightBlock(nsec, root-of-trust.Header.Height)
 - Expected precondition
-    - *FullNodes* is nonempty
+    - *FullNodes \ Secondaries* is nonempty
```

Comment:
As in the other PR, [LC-DATA-PEERS.1] defines that FullNodes and Secondaries are disjoint sets. It is better to clarify this than to do this change, I believe.